rust-create-cascade/src/main.rs

```diff
@@ -325,6 +326,9 @@ fn create_cascade(out_file: &Path, revoked_dir: &Path, known_dir: &Path, hash_al
         return;
     }
 
+    info!("Verifying cascade");
+    check_all(&cascade, revoked_dir, known_dir);
```

Comment:
I meant additionally checking the cascade that gets read from `cascade_bytes`, in the `if` block, but it's probably not a big deal.

Reply:
Ack, that's what I meant to do. Good catch!